packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -1,145 +1,493 @@
+        {
+          color: t.colors.brandForeground1,
+          colorDark: '#62ABF5',
+        },
+        t,
+      ),
     },
     brand: {
       filled: {
-        backgroundColor: t.colors.brandBackgroundStatic,
-        color: t.colors.neutralForegroundOnBrand,
-        iconColor: t.colors.neutralForegroundOnBrand,
+        ...getFilledColorProps(
+          {
+            backgroundColor: t.colors.brandBackgroundStatic,
+          },
+          t,
+        ),
       },
       outline: {
-        color: t.colors.brandForeground1,
-        borderColor: t.colors.brandForeground1,
-        iconColor: t.colors.brandForeground1,
+        ...getOutlineColorProps({ color: t.colors.brandForeground1, colorDark: '#62ABF5' }, t),
       },
       tint: {
-        backgroundColor: globalTokens.color.brand.tint60,
-        color: t.colors.brandForeground1,
-        borderColor: t.colors.brandStroke2,
-        iconColor: t.colors.brandForeground1,
+        ...getTintColorProps(
+          {
+            backgroundColor: globalTokens.color.brand.tint60,
+            color: t.colors.brandForeground1,
+            borderColor: t.colors.brandStroke2,
+            backgroundColorDark: globalTokens.color.outlook.shade40,
+            colorDark: '#77B7F7',
+            borderColorDark: globalTokens.color.outlook.shade40,
+          },
+          t,
+        ),
+      },
+      ghost: {
+        ...getGhostColorProps(
+          {
+            color: t.colors.brandForeground1,
+            colorDark: '#62ABF5',
+          },
+          t,
+        ),
       },
     },
     danger: {
       filled: {
-        backgroundColor: globalTokens.color.red.primary,
-        color: t.colors.neutralForegroundOnBrand,
-        iconColor: t.colors.neutralForegroundOnBrand,
+        ...getFilledColorProps(
+          {
+            backgroundColor: globalTokens.color.red.primary,
+          },
+          t,
+        ),
       },
       outline: {
-        color: globalTokens.color.red.primary,
-        borderColor: globalTokens.color.red.primary,
-        iconColor: globalTokens.color.red.primary,
+        ...getOutlineColorProps({ color: globalTokens.color.red.primary, colorDark: '#DC5F63' }, t),
       },
       tint: {
-        backgroundColor: globalTokens.color.red.tint60,
-        color: globalTokens.color.red.primary,
-        borderColor: '#F1BBBD',
-        iconColor: globalTokens.color.red.primary,
+        ...getTintColorProps(
+          {
+            backgroundColor: globalTokens.color.red.tint60,
+            color: globalTokens.color.red.primary,
+            borderColor: '#F1BBBD',
+            backgroundColorDark: globalTokens.color.red.shade40,
+            colorDark: globalTokens.color.red.tint30,
+            borderColorDark: globalTokens.color.red.shade30,
+          },
+          t,
+        ),
+      },
+      ghost: {
+        ...getGhostColorProps(
+          {
+            color: globalTokens.color.red.primary,
+            colorDark: '#DC5F63',
```

Comment:
>DC5F63 [](http://example.com/codeflow?start=25&length=6)

Please update these colors to use global tokens

Reply:
yeah, I'll do. I was waiting for confirmation from Daisy about the colors